.github/workflows/cpp-post-commit.yaml

```diff
@@ -68,6 +68,7 @@ jobs:
 
           {name: ttnn cpp tests, cmd: ./build/test/ttnn/unit_tests_ttnn},
           {name: ttnn ccl cpp unit tests, cmd: ./build/test/ttnn/unit_tests_ttnn_ccl},
+          {name: ttnn ccl cpp unit tests, cmd: ./build/test/ttnn/unit_tests_ttnn_tensor},
```

Comment:
make the pipeline name unique here
ttnn tensor cpp unit tests?

Reply:
ah is that why it fails maybe?